### ISSUE TITLE
Windows requirement

### ISSUE THREAD

**Issue by The-JMFB** [tarkah/lazystream#40]
Just a quick FYI for Windows users:

The Microsoft Visual C++ Redistributable  for VS 2XXXX is required for this binary to work, else you get:

The Application was Unable to Start Correctly (0xc000007b)

You can download from MS:  https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

Feel free to close this "issue", I just wanted to share in case anyone else happens to try and use this on a fresh Windows install.

**Comment by tarkah** [comment]
Thanks for opening this issue! I guess I already had those installed on my windows machines so I never realized :P 

I'll dig into this to see if this is something related to a dependency or if this affects most binaries built with Rust.

**Comment by The-JMFB** [comment]
Honestly, if you've installed many apps on your machine you will very likely already have it. Most applications I've used that require it will script the download/install with the application installer (off the top of my head, Kodi used to do that. Not sure if they still do as I haven't installed it for several years now). 

Only noticed it because I happened to be building a new media VM and I was testing my script for this before I had installed hardly anything else.

BTW - thanks very much for this. It's amazing to be able to grab MLB games for my family without needing to manually intervene EVERY SINGLE DAY FOR 8 OUT OF 12 MONTHS! LOL
